src/services/mouse.js

```diff
@@ -57,6 +57,12 @@ Controller.open(function(_) {
       e.preventDefault(); // doesn't work in IE≤8, but it's a one-line fix:
       e.target.unselectable = true; // http://jsbin.com/yagekiji/1
 
+      // if textarea is not focusable so the controller should not look for
```

Comment:
@jack-vo would recommend `//if the textarea does not have focus (e.g. because it is outside the viewport), the controller should not add a cursor to it`

is(':focus') returns true if the textarea currently has focus not whether it is focusable.